config/eslint.js

```diff
@@ -0,0 +1,13 @@
+// We need to specify the import/resolver setting in an extendable
```

Comment:
I'm surprised this works - rules later in the config overrode earlier ones - it's how you can enable a rule in a config, then disable it in your local eslint config. I would have thought settings would work in the same way.

Reply:
I think the import plugin iterates over the setting object, which gets merged together by ESLint's config. However, it merges in the order of plugin addition, and then iterates over the object (iteration happens in insertion order), so resolvers added by earlier plugins are actually handled first.